examples/iam-role-for-service-accounts-eks/main.tf

```
@@ -136,6 +136,22 @@ module "efs_csi_irsa_role" {
   tags = local.tags
 }
 
+module "s3_csi_irsa_role" {
+  source = "../../modules/iam-role-for-service-accounts-eks"
+
+  role_name            = "s3-csi"
+  attach_s3_csi_policy = true
+
+  oidc_providers = {
+    ex = {
+      provider_arn               = module.eks.oidc_provider_arn
+      namespace_service_accounts = ["kube-system:s3-csi-controller-sa"]
```

Per https://github.com/awslabs/mountpoint-s3-csi-driver/blob/3fd288dc112b9777e244b633ea12cd3f14ca581b/docs/install.md?plain=1#L74

```suggestion
      namespace_service_accounts = ["kube-system:s3-csi-driver-sa"]
```

---

udpated :)